v03_pipeline/lib/annotations/shared.py

```diff
@@ -98,10 +72,12 @@
     'unprocessed_pseudogene',
     'artifact',
     'lincRNA',
+    'lincrna',
```

Comment:
I... am not sure fi we expect other `RNA` case sensitivity issues.  I think we'll find out soon enough.